configure.ac

```diff
@@ -18,7 +18,7 @@ AC_CANONICAL_HOST
 AM_SILENT_RULES([yes])
 
 # Checks for libraries.
-PKG_CHECK_MODULES(OPENSSL, [libcrypto libssl])
+PKG_CHECK_MODULES(OPENSSL, [libcrypto >= 0.9.8 libssl >= 0.9.8], [], [AC_MSG_ERROR([Cannot find OpenSSL 0.9.8 or better.])])
```

Comment:
maybe "0.9.8 or higher" sounds better? apart from that it looks good to me. I didn't test if it actually solves #304 but I assume you have already verified that